bedrock/foundation/templates/foundation/annualreport/2017/index.html

```diff
@@ -11,6 +11,8 @@
   {{ css_bundle('annual_2017') }}
 {% endblock %}
 
+{% block body_class %}mzp-t-mozilla{% endblock %}
```

Comment:
Just an idea - rather than adding this explicitly to each template that needs it, I wonder if it would be worth making a `base-protocol-mozilla.html` template, which pages could inherit from, as opposed to overriding the body class each time? Maybe feels a bit more DRY?

Reply:
It does feel more DRY. If I add it using the body_class block then it can be over-ridden by other body_class declarations. Do you think that is desirable behavoiur or should I find a way to declare it in the `base-protocol-mozilla.html` template that cannot be over-ridden?

Reply:
I’m happy either way to be honest. If we think that this is a solid rule that won’t change, your suggestion of hardcoding it is a good one? 🙂

Reply:
Turns out I don't know how to hard code it without duplicating `base-pebbles.html` which seemed like a bad idea. So we just have a simple `block body_class` declaration. The templates are separate now though so it could be easy to change how they work later.

Reply:
I made the changes to the "Add Mozilla theme to Mozilla pages" commit instead of adding them as review fixes. Sorry if it complicates things for reviewing.